statsmodels/tsa/arima_process.py

```diff
@@ -108,13 +73,14 @@ def arma_generate_sample(ar, ma, nsample, sigma=1, distrvs=np.random.randn,
     >>> model.params
     array([ 0.79044189, -0.23140636,  0.70072904,  0.40608028])
     """
-    #TODO: unify with ArmaProcess method
-    eta = sigma * distrvs(nsample+burnin)
+    # TODO: unify with ArmaProcess method
+    eta = sigma * distrvs(nsample + burnin)
```

Comment:
In general (especially at the time I wrote these kind of things)  
A distribution always refers to scipy.stats.distribution and compatible with that interface. Otherwise, a user needs to use a lambda function or similar to get a compatible function, e.g. I think in this case it can be any function that has already partialled out the distribution parameters and just needs the number of observations that should be created. For example the rvs method of a `frozen` distribution from scipy.stats should work.